pytorch_lightning/utilities/distributed.py

```diff
@@ -65,22 +65,26 @@ def _get_rank() -> int:
 rank_zero_only.rank = getattr(rank_zero_only, 'rank', _get_rank())
 
 
-def _warn(*args, **kwargs):
-    warnings.warn(*args, **kwargs)
-
-
-def _info(*args, **kwargs):
+def _info(*args, stacklevel: int = 2, **kwargs):
+    if python_version() >= "3.8.0":
```

Comment:
can we add docstring here or at least refer `stacklevel` meaning/usage?

Reply:
It's part of the Python logging API, but there's not a direct link to reference

Reply:
I would say adding this link is good https://github.com/PyTorchLightning/pytorch-lightning/pull/8002#discussion_r655528895

Reply:
That doesn't link to the stacklevel description exactly, and you wouldn't want to link to the debug docs also for `info` and `warn`